xtask/src/commands/release/semver_check.rs

```diff
@@ -169,6 +177,9 @@ pub mod checker {
         }
 
         if !semver_incompatible_packages.is_empty() {
+            // let dest_path = workspace.join("esp-rom-sys/src/generated_rom_symbols.rs");
```

Comment:
commented code always looks sus ... especially without any additional comment

I guess this is just a left-over